src/components/Article.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+function Article(props) {
```

Comment:
делай деструктуризацию props
```suggestion
function Article({ article }) {
```

Reply:
<img width="438" alt="image" src="https://user-images.githubusercontent.com/38065883/80089452-a2a37080-8566-11ea-866b-4220aafd29b7.png">
Компонент должен принимать три props